app/assets/javascripts/spree/backend/spree_volume_pricing.js

```diff
@@ -14,3 +14,76 @@ $(function () {
   });
 
 });
+
+$(document).ready(function () {
```

Comment:
We want to support turbolinks so this won't work

Reply:
@brettallred spree don't use turbolinks in backend  layout